tasks/vm-aws.yml

```diff
@@ -96,6 +102,7 @@
       Project: "{{ cornerstone_prefix }}"
       Purpose: "{{ cornerstone_tag_purpose }}"
       Role: "{{ cornerstone_tag_role }}"
+      kubernetes.io/cluster/openshift: "clusterid"
```

Comment:
Slightly confused why this is here? We probably ought to come up with a better way of adding tags generally.